django-verdant/rca_ee/models.py

```diff
@@ -107,10 +107,16 @@ def serve(self, request):
         context['form'] = form
         return render(
             request,
-            self.template,
+            self.get_template(request),
             context
         )
 
+    def get_template(self, request, *args, **kwargs):
+        if request.is_ajax() and request.GET.get('format') == 'enquiry_form':
```

Comment:
`/some-index?page=1` and `/some-index?page=2` are treated separately by ATS, but I've just realised that the ajax form needs a GET param too, so you're right, the vary header is not strictly necessary here in `rca_ee.models.FormPage`.